src/test/ui/const-generics/min_const_generics/impl-trait-with-const-arguments.rs

```diff
@@ -0,0 +1,22 @@
+#![feature(min_const_generics)]
```

Comment:
this test has the same output with `const_generics` afaict, was there a reason why you didn't use revisions for that one?

Reply:
Just an oversight; I'll correct it.